pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java

```diff
@@ -223,7 +226,7 @@ public static Iterable<Object[]> configsWithErrors() {
         result.add(new Object[]{"invalidWrapperType", "Invalid option type \"echo\". Valid option types:"});
         result.add(new Object[]{"invalidStageWrapperType", "Invalid option type \"echo\". Valid option types:"});
 
-        result.add(new Object[]{"unknownBareAgentType", Messages.ModelValidatorImpl_InvalidAgentType("foo", legalAgentTypes)});
+        result.add(new Object[]{"unknownBareAgentType", Messages.ModelValidatorImpl_InvalidAgentType("foo", "[any, label, none, otherField]")});
```

Comment:
Newly consistent with `unknownAgentType` in the sense that both now hardcode a list of agent types rather than using `legalAgentTypes`, which is not yet initialized when this code is running.